Nakama/Source/NakamaCore/Public/nakama-cpp/NClientInterface.h

```diff
@@ -20,6 +20,7 @@
 #include <functional>
```

Comment:
this file as a whole seems to be syncrhonised with nakama-cpp, can it be a simple reinclude of a file from `vcpkg_installed` ? Or better add real vcpkg nakama-cpp headers to the search path in `NakamaCore.build.cs` if possible and don't have a copy of it here at all?

Reply:
Yeah I agree with you that we shouldn't be putting the libraries and headers in source. I've filed an issue to improve this later because there isn't time at the moment (would have to rework how we distribute the plugin.)